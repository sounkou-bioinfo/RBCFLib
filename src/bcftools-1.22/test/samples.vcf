##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##bcftoolsVersion=1.21-131-g05621cf-dirty+htslib-1.21-59-g45e1390-dirty
##bcftoolsCommand=mpileup -Ou -f samples.fa samples.sam
##reference=file://samples.fa
##contig=<ID=NC_000074.7:89373943-89415103,length=41161>
##ALT=<ID=*,Description="Represents allele(s) other than observed.">
##INFO=<ID=INDEL,Number=0,Type=Flag,Description="Indicates that the variant is an INDEL.">
##INFO=<ID=IDV,Number=1,Type=Integer,Description="Maximum number of raw reads supporting an indel">
##INFO=<ID=IMF,Number=1,Type=Float,Description="Maximum fraction of raw reads supporting an indel">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Raw read depth">
##INFO=<ID=VDB,Number=1,Type=Float,Description="Variant Distance Bias for filtering splice-site artefacts in RNA-seq data (bigger is better)",Version="3">
##INFO=<ID=RPBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Read Position Bias (closer to 0 is better)">
##INFO=<ID=MQBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Mapping Quality Bias (closer to 0 is better)">
##INFO=<ID=BQBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Base Quality Bias (closer to 0 is better)">
##INFO=<ID=MQSBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Mapping Quality vs Strand Bias (closer to 0 is better)">
##INFO=<ID=SCBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Soft-Clip Length Bias (closer to 0 is better)">
##INFO=<ID=SGB,Number=1,Type=Float,Description="Segregation based metric, http://samtools.github.io/bcftools/rd-SegBias.pdf">
##INFO=<ID=MQ0F,Number=1,Type=Float,Description="Fraction of MQ0 reads (smaller is better)">
##INFO=<ID=I16,Number=16,Type=Float,Description="Auxiliary tag used for calling, see description of bcf_callret1_t in bam2bcf.h">
##INFO=<ID=QS,Number=R,Type=Float,Description="Auxiliary tag used for calling">
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="List of Phred-scaled genotype likelihoods">
##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths (high-quality bases)">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample1	sample2	sample3
NC_000074.7:89373943-89415103	1	.	G	<*>	0	.	DP=6;I16=6,0,0,0,204,6936,0,0,360,21600,0,0,0,0,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,34:1,0	0,6,62:2,0	0,9,85:3,0
NC_000074.7:89373943-89415103	2	.	G	<*>	0	.	DP=6;I16=6,0,0,0,108,1944,0,0,360,21600,0,0,6,6,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,18:1,0	0,6,33:2,0	0,9,44:3,0
NC_000074.7:89373943-89415103	3	.	T	<*>	0	.	DP=6;I16=6,0,0,0,198,6534,0,0,360,21600,0,0,12,24,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,33:1,0	0,6,60:2,0	0,9,82:3,0
NC_000074.7:89373943-89415103	4	.	G	<*>	0	.	DP=6;I16=6,0,0,0,240,9600,0,0,360,21600,0,0,18,54,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,40:1,0	0,6,73:2,0	0,9,100:3,0
NC_000074.7:89373943-89415103	5	.	G	<*>	0	.	DP=6;I16=6,0,0,0,198,6534,0,0,360,21600,0,0,24,96,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,33:1,0	0,6,60:2,0	0,9,82:3,0
NC_000074.7:89373943-89415103	6	.	G	<*>	0	.	DP=6;I16=6,0,0,0,252,10584,0,0,360,21600,0,0,30,150,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,42:1,0	0,6,77:2,0	0,9,105:3,0
NC_000074.7:89373943-89415103	7	.	T	<*>	0	.	DP=6;I16=6,0,0,0,198,6534,0,0,360,21600,0,0,36,216,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,33:1,0	0,6,60:2,0	0,9,82:3,0
NC_000074.7:89373943-89415103	8	.	G	<*>	0	.	DP=6;I16=6,0,0,0,210,7350,0,0,360,21600,0,0,42,294,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,35:1,0	0,6,64:2,0	0,9,87:3,0
NC_000074.7:89373943-89415103	9	.	G	<*>	0	.	DP=6;I16=6,0,0,0,162,4374,0,0,360,21600,0,0,48,384,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,27:1,0	0,6,49:2,0	0,9,67:3,0
NC_000074.7:89373943-89415103	10	.	G	<*>	0	.	DP=6;I16=6,0,0,0,234,9126,0,0,360,21600,0,0,54,486,0,0;QS=3,0;MQ0F=0	PL:AD	0,3,39:1,0	0,6,71:2,0	0,9,97:3,0
