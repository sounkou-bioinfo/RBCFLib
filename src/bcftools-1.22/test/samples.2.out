##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##bcftoolsVersion=1.21-131-g05621cf-dirty+htslib-1.21-59-g45e1390-dirty
##bcftoolsCommand=mpileup -Ou -f samples.fa samples.sam
##reference=file://samples.fa
##contig=<ID=NC_000074.7:89373943-89415103,length=41161>
##ALT=<ID=*,Description="Represents allele(s) other than observed.">
##INFO=<ID=INDEL,Number=0,Type=Flag,Description="Indicates that the variant is an INDEL.">
##INFO=<ID=IDV,Number=1,Type=Integer,Description="Maximum number of raw reads supporting an indel">
##INFO=<ID=IMF,Number=1,Type=Float,Description="Maximum fraction of raw reads supporting an indel">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Raw read depth">
##INFO=<ID=VDB,Number=1,Type=Float,Description="Variant Distance Bias for filtering splice-site artefacts in RNA-seq data (bigger is better)",Version="3">
##INFO=<ID=RPBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Read Position Bias (closer to 0 is better)">
##INFO=<ID=MQBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Mapping Quality Bias (closer to 0 is better)">
##INFO=<ID=BQBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Base Quality Bias (closer to 0 is better)">
##INFO=<ID=MQSBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Mapping Quality vs Strand Bias (closer to 0 is better)">
##INFO=<ID=SCBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Soft-Clip Length Bias (closer to 0 is better)">
##INFO=<ID=SGB,Number=1,Type=Float,Description="Segregation based metric, http://samtools.github.io/bcftools/rd-SegBias.pdf">
##INFO=<ID=MQ0F,Number=1,Type=Float,Description="Fraction of MQ0 reads (smaller is better)">
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="List of Phred-scaled genotype likelihoods">
##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths (high-quality bases)">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes for each ALT allele, in the same order as listed">
##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
##INFO=<ID=DP4,Number=4,Type=Integer,Description="Number of high-quality ref-forward , ref-reverse, alt-forward and alt-reverse bases">
##INFO=<ID=MQ,Number=1,Type=Integer,Description="Average mapping quality">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample3
NC_000074.7:89373943-89415103	1	.	G	.	114.588	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	2	.	G	.	73.587	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	3	.	T	.	111.587	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	4	.	G	.	129.588	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	5	.	G	.	111.587	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	6	.	G	.	134.588	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	7	.	T	.	111.587	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	8	.	G	.	116.588	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	9	.	G	.	96.5873	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
NC_000074.7:89373943-89415103	10	.	G	.	126.588	.	DP=6;MQ0F=0;AN=2;DP4=6,0,0,0;MQ=60	GT:AD	0/0:3
